ckan/logic/action/update.py

```diff
@@ -1308,3 +1311,87 @@ def bulk_update_delete(context, data_dict):
 
     _check_access('bulk_update_delete', context, data_dict)
     _bulk_update_dataset(context, data_dict, {'state': 'deleted'})
+
+
+def config_option_update(context, data_dict):
+    '''
+
+    .. versionadded:: 2.4
+
+    Allows to modify some CKAN configuration options
```

Comment:
I think this is clearer as: "Modify the values of one or more CKAN configuration options"